code-quality/accessibility/README.md

```diff
@@ -0,0 +1,74 @@
+# Accessibility
+
+## What is it?
+
+Accessibility is basically making sure web sites care fully inclusive and can be properly access for **everyone**, not just people with disabilities.
```

Comment:
`....websites are fully...`

Reply:
`...can be properly accessed...`

Reply:
Maybe change this verbiage a bit to 
`...not only people with disabilities, but everyone.`

Reply:
Went with 'properly accessed by..`, otherwise gonna do these changes.